trpc/routers/_app.ts

```
@@ -1,30 +1,41 @@
-
-    if (!userId) {
-      return {
-        greeting: "Hello! You are not signed in.",
-      };
-    }
 
-    return {
-      greeting: `Hello ${userId}!`,
-    };
+export const appRouter = createTRPCRouter({
+  allUsers: baseProcedure.query(async (opts) => {
```

ESLint

[error] 'opts' is defined but never used.

(@typescript-eslint/no-unused-vars)